mail/templates/product_order_receipt/body.html

```diff
@@ -55,12 +55,19 @@ <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px;
                         </tr>
                     </table>
                 {% endif %}
+                <strong style="font-weight: 700;">Email Address:</strong> {{ purchaser.email }}<br>
             </p>
             {% if receipt or coupon %}
                 <h3 style="color: #000000; font-size: 16px; font-weight: 700; line-height: 18px; margin: 0 0 20px;">Payment Information</h3>
                 <p>
                     {%  if receipt %}
-                        <strong style="font-weight: 700;">Payment Method:</strong> {{ receipt.card_type }} | {{ receipt.card_number }}<br>
+                        {% if receipt.payment_method == 'card' %}
+                            <strong style="font-weight: 700;">Name:</strong> {{ receipt.name }}<br>
```

Comment:
Should not be the `name` independent to payment_method ? I mean, name should not be out of the `if` conditions here ?

Reply:
sure, moving it out of it.